deploy/charts/openebs-monitoring/dashboards/Control plane/openebs-main-dashboard.json

```diff
@@ -0,0 +1,2187 @@
+          "defaults": {
+            "custom": {
+              "align": "center"
+            },
+            "mappings": [
+              {
+                "from": "",
+                "id": 0,
+                "operator": "",
+                "text": "Up",
+                "to": "",
+                "type": 1,
+                "value": "1"
+              },
+              {
+                "from": "",
+                "id": 1,
+                "operator": "",
+                "text": "Down",
+                "to": "",
+                "type": 1,
+                "value": "0"
+              }
+            ],
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "green",
+                  "value": null
+                },
+                {
+                  "color": "red",
+                  "value": 80
+                }
+              ]
+            }
+          },
+          "overrides": [
+            {
+              "matcher": {
+                "id": "byName",
+                "options": "phase"
+              },
+              "properties": [
+                {
+                  "id": "custom.width",
+                  "value": 88
+                },
+                {
+                  "id": "custom.displayMode",
+                  "value": "color-text"
+                },
+                {
+                  "id": "mappings",
+                  "value": [
+                    {
+                      "from": "",
+                      "id": 1,
+                      "operator": "",
+                      "text": "",
+                      "to": "",
+                      "type": 1
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "matcher": {
+                "id": "byName",
+                "options": "namespace"
+              },
+              "properties": [
+                {
+                  "id": "custom.width",
+                  "value": 200
+                }
+              ]
+            }
+          ]
+        },
+        "gridPos": {
+          "h": 7,
+          "w": 11,
+          "x": 0,
+          "y": 20
+        },
+        "id": 30,
+        "links": [],
+        "options": {
+          "showHeader": true,
+          "sortBy": [
+            {
+              "desc": false,
+              "displayName": "persistentvolumeclaim"
+            }
+          ]
+        },
+        "pluginVersion": "7.1.1",
+        "targets": [
+          {
+            "expr": "kube_persistentvolumeclaim_status_phase{cluster=\"$cluster\"}",
+            "format": "table",
+            "instant": true,
+            "interval": "",
+            "legendFormat": "{{persistentvolumeclaim}}:{{phase}}",
+            "refId": "A"
+          }
+        ],
+        "timeFrom": null,
+        "timeShift": null,
+        "title": "persistentvolumeclaim in namespace $namespace",
```

Comment:
This panel displays pvc's that are present inside other ns as present inside `openebs` ns. Please check and display only pvc that are present only inside `openebs` ns.